src/Effect.elm

```diff
@@ -5,6 +5,7 @@ port module Effect exposing
     , pushUrlPath
     , save
     , toCmd
```

Comment:
maybe a name like `promptUnsavedChanges`, `warnUnsavedChanges` would be more descriptive than `promptToSave`